templates/content/taxonomy-term--localgov-blog-author.html.twig

```diff
@@ -25,13 +25,12 @@
 #}
 {% if not localgov_base_remove_css %}
   {{ attach_library('localgov_base/term') }}
-  {#{{ attach_library('localgov_base/blog') }}#}
 {% endif %}
 
 <div{{ attributes }}>
-  <div class="lgd-term--content">
+  <div class="lgd-term">
```

Comment:
I think we can remove this, if we add the class to the parent (or leave it here, and we won't bother adding it to the parent/attributes)